crawl-ref/source/dat/des/branches/orc.des

```diff
@@ -1399,6 +1399,433 @@ x1...1...1x
+ ..H...F...xxx
+ ..H.O.xFFxx
+   x......x
+   x...O..x
+   xxxxxxxx
+ENDMAP
+
+NAME:   nicolae_orc_quarried_blocks
+TAGS:   transparent no_exits
+DEPTH:  Orc
+NSUBST: @ = @ / 2=@___ / _
+SUBST:  D : xxxx_, E : xxxx_, F : xxxx_, H : xxxx_, J : xxxx_, K : xxxx_
+SUBST:  L : xxxx_, M : xxxx_, N : xxxx_, O : xxxx_, P : xxxx_, Q : xxxx_
+SUBST:  R : xxxx_, S : xxxx_, Z : xxxx_, - : xxxx_
+CLEAR:  _
+MAP
+DDD EEE FFF HHH JJJ
+DDD EEE FFF HHH JJJ
+DDD EEE FFF HHH JJJ
+       @   @
+KKK xxx xxx xxx LLL
+KKK xxx xxx xxx LLL
+KKK xxx xxx xxx LLL
+   @   @   @   @
+MMM xxx xxx xxx NNN
+MMM xxx xxx xxx NNN
+MMM xxx xxx xxx NNN
+   @   @   @   @
+OOO xxx xxx xxx PPP
+OOO xxx xxx xxx PPP
+OOO xxx xxx xxx PPP
+       @   @
+QQQ RRR SSS ZZZ ---
+QQQ RRR SSS ZZZ ---
+QQQ RRR SSS ZZZ ---
+ENDMAP
+
+# I don't want to be buried / in an orc seminary
+# wait that's not how the song goes
+NAME:  nicolae_orc_seminary
+TAGS:  transparent
+DEPTH: Orc
+KITEM: _ = manual of invocations w:3 / amulet of faith ident:all w:12
+KMONS: _ = orc high priest
+KMONS: P = orc priest
+KFEAT: _ = altar_beogh
+SUBST: P = P .:15
+MAP
+ccccccccc
+cc.P.P.cc
+cI.....Pc
+c..P.P..+
+c_......+
+c..P.P..+
+cI.....Pc
+cc.P.P.cc
+ccccccccc
+ENDMAP
+
+# St. Roka, don't you call me, 'cause I can't go,
+NAME:   nicolae_orc_sixteen_tons
+TAGS:   transparent patrolling
+DEPTH:  Orc
+KITEM:  r = large rock q:1
+KMONS:  O = spectral orc w:8 / spectral orc warrior w:4 / \
+            spectral orc knight w:2 / spectral orc warlord w:1
+KFEAT:  S = general shop type:Company suffix:Store
+SUBST:  - = .__
+NSUBST: . = 16=r / 4=O / 4=O. / .
+CLEAR:  _
+MAP
+   -----
+  -.....-
+ -.......-
+-.........-
+-.........-
+-....S....-
+-.........-
+-.........-
+ -.......-
+  -.....-
+   -----
+ENDMAP
+
+# yes i know that's not a word
+NAME:  nicolae_orc_stalacmites
+TAGS:  transparent
+DEPTH: Orc
+SUBST: ' = x.
+MAP
+xx        xxx        xx
+'xxx    xxx'xxx    xxx'
+.x'xxxxxx'x.x'xxxxxx'x.
+.x.x'xx'x.x.x.x'xx'x.x.
+.x.x.''.x.x.x.x.''.x.x.
+.x.'....'.x.x.'....'.x.
+.'........'.'........'.
+@.....................@
+'..........'..........'
+x.'......'.x.'......'.x
+x.x.'..'.x.x.x.'..'.x.x
+x.x.x''x.x.x.x.x''x.x.x
+x.x'xxxx'x.x.x'xxxx'x.x
+x'xxx  xxx'x'xxx  xxx'x
+xxx      xxxxx      xxx
+ENDMAP
+
+# Boss makes a dollar / I make a dime / That's why I go / On company time
+NAME:  nicolae_orc_takin'_care_of_business
+TAGS:  transparent
+DEPTH: Orc, !Orc:$
+KITEM: % = any scroll
+KMONS: %1 = patrolling orc / nothing w:15
+KFEAT: ^ = shaft trap
+MAP
+xxxxxxxxxx
+x^%x^%x^%x
+x..x..x..x
+xx+xx+xx+x
+.1..1..1..
+ENDMAP
+
+NAME:    nicolae_orc_troll_toll
+TAGS:    transparent
+DEPTH:   Orc
+KFEAT:   _ = altar_beogh
+KMONS:   D = deep troll
+KMONS:   R = troll
+# 1/4 chance of having all three staircases.
+SUBST:   ( = GGGII_<<<(((, R = R...
+SHUFFLE: {[(
+: if you.depth() == 1 then
+SUBST:   { = }, [ = ], ( = ), < = >
+: end
+MAP
+ccccc
+c...c       ...
+cR{.cccccccc.R..
+c...+.R.R.Rc.....
+cD(.+R.R.R.+..R..
+c...+.R.R.Rc.....
```

Comment:
Adding slightly more width to the stairs section so the deep troll isn't directly beside the stairs when one goes down possibly the only non-teleport / dig staircase to Orc:2 is probably for the better. Even the most dangerous of the various ambush vaults don't place the threats directly adjacent to the stairs.